invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/PatronCurrentLoans/__tests__/PatronCurrentLoans.js

```diff
@@ -27,7 +27,7 @@ describe('PatronLoans tests', () => {
     const component = shallow(
       <PatronCurrentLoans
         history={() => {}}
-        data={[]}
+        data={{}}
```

Comment:
Maybe `data={}` is enough

Reply:
`data` is marked as a required prop in `PatronCurrentLoans` and JSX will complain if we pass nothing. But maybe `data` shouldn't be a required prop in any of these components at all? If we keep `data` optional and update all components to work with empty data it would fix #225.